.github/workflows/aws-deploy.yml

```diff
@@ -6,16 +6,17 @@ name: Deploy to AWS
 on:
   workflow_run:
     workflows: [ "validate" ]
-    branches: [main]
+    branches: ['dev', 'prod']
```

Comment:
Is the `validate` workflow run when merging into `main`?  (Seems like we want the static analysis and unit tests to be run before we decide to deploy to dev or prod.)